provisioning/setup.yaml

```diff
@@ -23,3 +20,5 @@ hare:
   prepare:
     cmd: /opt/seagate/cortx/hare/bin/hare_setup prepare
     args: --config $URL
+support_bundle:
```

Comment:
Looks here too needs some indentations correcion

Reply:
No, indentation is correct, support_bundle key is not expected to be a child of hare key as per the setup.yaml template mentioned in the commit message.